test/dlc_tests/sanity/quick_checks/test_prepare_dev_env.py

```diff
@@ -0,0 +1,166 @@
+import pytest
+
+from src import prepare_dlc_dev_environment
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("build_frameworks")
+def test_build_frameworks():
+    overrider = prepare_dlc_dev_environment.TomlOverrider()
+    overrider.set_build_frameworks(("pytorch", "tensorflow"))
+
+    assert overrider.overrides == {"build": {"build_frameworks": ["pytorch", "tensorflow"]}}
```

Comment:
looks like these are going to have to be more specific. you can do something like 

```
assert overrider.overrides["build"]["build_frameworks"] == ["pytorch", "tensorflow"]
```

and apply this same pattern to the other failing tests below, making sure to swap out the keys and values for their expected results

example for job types

```
assert overrider.overrides["build"]["build_training"] == True and overrider.overrides["build"]["build_inference"] == True 
```